ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/minimal-permission/provision/cucushift-installer-rehearse-gcp-ipi-minimal-permission-provision-chain.yaml

```diff
@@ -2,6 +2,7 @@ chain:
   as: cucushift-installer-rehearse-gcp-ipi-minimal-permission-provision
   steps: 
     - ref: gcp-provision-minimal-permission
+    - ref: gcp-provision-bastionhost
```

Comment:
I do not think this works, because the bastionhost need to provision customer vpc as prerequisite. Why not use an existing install workflow which already includes bastion host to run your testing, e.g: proxy, private, disconnected install workflows, all of them already provision customer vpc and bastion host in the install workflow.

Reply:
We have a plan to support DR test on some test jobs which don't provide bastion host, detail see https://issues.redhat.com/browse/OCPQE-22058?focusedId=24751181&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-24751181, some of our DR test cases need bastion host, otherwise won't be executed correctly. We have add DR test to proxy, private, disconnected install workflows.

Reply:
@jianlinliu Do you think that running DR tests whitout provision customer vpc as prerequisite doesn't make sense?

Reply:
cc: @geliu2016 @wsun1

Reply:
Here your testing requires bastion host, but for where to place the bastion on the cloud?  then need to provision vpc in advance, that is the "prerequisite" for bastion host.

Reply:
Oaky, I see, closing this PR.